library/vector.lisp

```diff
@@ -197,7 +197,7 @@
 
   (declare foreach2 ((:a -> :b -> :c) -> Vector :a -> Vector :b -> Unit))
   (define (foreach2 f v1 v2)
-    "Like vector-foreach but twice as good"
+    "Iterate in parallel over V1 and V2 calling F once for each pair of elements. Iteration stops when the first vector ends"
```

Comment:
Should this be "Iteration stops when the shorter of the two vectors runs out of elements" or something to that effect?